x-pack/legacy/plugins/monitoring/server/lib/setup/collection/get_collection_status.js

```diff
@@ -316,8 +316,8 @@ export const getCollectionStatus = async (req, indexPatterns, clusterUuid, nodeU
   ]);
 
   const liveClusterUuid = skipLiveData ? null : await getLiveElasticsearchClusterUuid(req);
-  const liveEsNodes = skipLiveData ? [] : await getLivesNodes(req);
-  const liveKibanaInstance = skipLiveData ? {} : await getLiveKibanaInstance(req);
+  const liveEsNodes = skipLiveData || (clusterUuid && liveClusterUuid !== clusterUuid) ? [] : await getLivesNodes(req);
+  const liveKibanaInstance = skipLiveData || (clusterUuid && liveClusterUuid !== clusterUuid) ? {} : await getLiveKibanaInstance(req);
```

Comment:
Just nitpicking here, but we can also put:
`skipLiveData || (clusterUuid && liveClusterUuid !== clusterUuid)` into its own const, so it's evaluated only once (and shortens line width)

Reply:
Yea we could, but I'm not convinced it's necessary if we're only repeating the code twice. Since this is already passing CI, I'm just going to go ahead and merge it. If you feel strongly about, feel free to open an issue and we can discuss more